### PR TITLE
Sidebar Changes, Font and Color Updates (Tool)

### PR DESCRIPTION
**Pull request by naveenrs123**
This PR contains the following changes:
SESSION[62547387ccb90d3958f4e488]_C[0]: [Changed font-size of %[116]% to 8rem.]
SESSION[62547387ccb90d3958f4e488]_C[1]: [Sidebar (%[34]%) is now 240px wide.]
SESSION[62547387ccb90d3958f4e488]_C[2]: [Added Twitter contact link to sidebar. %[94]%]
SESSION[62547387ccb90d3958f4e488]_C[3]: [Changed the "Link to Project" text %[549]% CSS. Now the main change is a color change.]

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -83,12 +83,17 @@
...
-        <div id="sidebar-info">
-          <a class="info-text" href="https://icons8.com"
-            >Icons by <strong>Icons8</strong></a
-          >
-        </div> -->
+        <div class="sidebar-contact">
+          <a
+            data-toggle="tooltip"
+            data-placement="top"
+            title="twitter"
```
Comment (randomion) [inline index.html:90]:
```suggestion
            title="Twitter"
```

File: index.html [hunk 2]
```diff
@@ -83,12 +83,17 @@
...
+            href="https://www.twitter.com/nsivasa/"
+            ><img
+              class="contact-img"
+              src="https://img.icons8.com/ios-filled/48/ececec/twitter.png"
+          /></a>
+        </div>
```
Comment (randomion) [inline index.html:96]: SESSION[62547b4fccb90d3958f4e489]_C[0]: [How about this layout instead?Add flex-wrap: wrap to the container and set the width of each element to 50%.]

File: index.html [hunk 3]
```diff
@@ -83,12 +83,17 @@
           /></a>
         </div>
 
-        <!--
-        <div id="sidebar-info">
-          <a class="info-text" href="https://icons8.com"
-            >Icons by <strong>Icons8</strong></a
-          >
-        </div> -->
```
Comment (randomion) [inline index.html:91]: Why are we not keeping this in the sidebar?

**Review by randomion — CHANGES_REQUESTED** [review]
A few changes needed but looks good in general.

File: styles.css [hunk 4]
```diff
@@ -207,7 +207,7 @@ nav {
 }
 
 .about-content h1 {
-  font-size: 6rem;
+  font-size: 8rem;
```
[the remark this answers is not in the document]
Comment (naveenrs123) [reply styles.css:210]: Noted. I will test other sizes and see what looks best.

File: index.html (hunk 1 above)
Comment (naveenrs123) [reply index.html:90]: Thanks, will update.

File: index.html (hunk 3 above)
Comment (naveenrs123) [reply index.html:91]: SESSION[62547387ccb90d3958f4e488]_C[1]: [Sidebar (%[34]%) is now 240px wide.]

The sidebar is already quite packed. With your suggestion in https://github.com/naveenrs123/pr-referencing-study-repo/pull/7#discussion_r847647448, the vertical space is further decreased. I don't want there to be scrolling issues on smaller screens.

**Comment by naveenrs123** [comment]
SESSION[6255feb70e00a753504bb418]_C[0]: [I am commenting on %[156]% as a test.]
SESSION[6255fee50e00a753504bb419]_C[1]: [This is a new session with a different reference. I am referencing %[473]% in this comment.]

**Comment by naveenrs123** [comment]
SESSION[6255dcebef36c087b4e5d1cd]_C[0]: [Updated text %[198]% in hobbies section %[196]%.]
SESSION[6255dcebef36c087b4e5d1cd]_C[1]: [Added icon attribution. %[94]%]
SESSION[6255dcebef36c087b4e5d1cd]_C[2]: [Change layout of the skills section %[226]% so that all 4 items fit on the same row.]
SESSION[6255dcebef36c087b4e5d1cd]_C[3]: [Change color of section titles from red to green e.g. %[223]% and %[268]%]
SESSION[6255dcebef36c087b4e5d1cd]_C[4]: [Change color of "Link to Project" %[504]% to green on hover.]

File: index.html (hunk 2 above)
Comment (naveenrs123) [reply index.html:96]: SESSION[6255fee50e00a753504bb419]_C[1]: [This is a new session with a different reference. I am referencing %[473]% in this comment.]

Test with this interface comment.

Comment (naveenrs123) [reply index.html:96]: SESSION[6255fee50e00a753504bb419]_C[1]: [This is a new session with a different reference. I am referencing %[473]% in this comment.]

Test with this interface comment.

Comment (naveenrs123) [reply index.html:96]: SESSION[6255fee50e00a753504bb419]_C[1]: [This is a new session with a different reference. I am referencing %[473]% in this comment.]

File: styles.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (naveenrs123) [reply styles.css:210]: SESSION[6255fee50e00a753504bb419]_C[1]: [This is a new session with a different reference. I am referencing %[473]% in this comment.]

**Pull request closed without merging**